_pages/software-and-tools/github.md

```diff
@@ -110,7 +110,11 @@ Contractors or external government collaborators should only be added to teams w
 
 ### Archiving
 
-As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date. Inactive repositories are automatically archived via [ghad](https://github.com/18F/ghad).
+As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive](https://help.github.com/articles/archiving-repositories/) repositories to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date.
```

Comment:
Nit: have the hyperlink include more text.

```suggestion
As discussed in the [18F open source policy](https://github.com/18F/open-source-policy/blob/master/practice.md), we [archive repositories](https://help.github.com/articles/archiving-repositories/) to deprecate them. In short, that means we are no longer maintaining them, including keeping dependencies up-to-date.
```

Reply:
It's a good nit, Bront